### PR TITLE
[etesync-server] add guide for EteSync Server

### LINKED ISSUE #710: EteSync Guide WIP
I set up [EteSync server](https://github.com/etesync/server/) on my Uberspace and will try to create a new guide for the installation process here: https://github.com/Nepoh/lab/tree/nepoh-etesync

This is my first attempt to create a guide for the lab, so feel free to contribute!

Roadmap:
- [x] convert my own notes to reST
- [x] check for completenes
- [x] test on a fresh uberspace
- [x] check the [Style Guide](https://github.com/Uberspace/lab/blob/master/STYLE.md)
- [x] find appropriate [tags](https://lab.uberspace.de/tags)
- [x] sum up & create pull request

### PR DESCRIPTION
**Pull request by Nepoh**
New guide for EteSync.

closes #710

### CODE REVIEW

File: source/guide_etesync-server.rst [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+ [isabell@stardust etesync_server]$
+
+Step 3
+------
+
+Apply database migrations:
+
+::
+
+ [isabell@stardust etesync_server]$ python3.6 manage.py migrate
```
[the remark this answers is not in the document]
Comment (Nepoh) [reply source/guide_etesync-server.rst]: I meant to keep things as simple as possible, but mysql is possible and probably makes sense => added

File: source/guide_etesync-server.rst [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+ 152 static files copied to '/home/isabell/etesync_server/static'.
+ [isabell@stardust etesync_server]$
+
+Configuration
+=============
+
+Configure Webserver
+-------------------
+
+Open ``~/etesync_server/etesync_server/settings.py`` and edit the line ``ALLOWED_HOSTS = []`` to add your host name (e.g. ``isabell.uber.space``):
```
[the remark this answers is not in the document]
Comment (Nepoh) [reply source/guide_etesync-server.rst]: Oh, I did not know that. This makes things a lot easyer and cleaner. But it needs to go here: `~/etesync_server/etesync_site_settings.py` where `manage.py` lives.

File: source/guide_etesync-server.rst [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+
+To deploy your application with ``uwsgi``, create a file at ``~/uwsgi/apps-enabled/etesync_server.ini`` with the following content:
+
+.. warning:: Replace ``<username>`` with your username! (3 times)
+
+.. code-block:: ini
+  :emphasize-lines: 2,15,16
+
+  [uwsgi]
+  chdir = /home/<username>/etesync_server
```
[the remark this answers is not in the document]
Comment (Nepoh) [reply source/guide_etesync-server.rst]: Nice, I'm new to uwsgi and did not know this was possible!

File: source/guide_etesync-server.rst [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+
+.. note:: Check the release_ page on GitHub regularly or follow the feed_ to stay informed about the newest version.
+
+If there is a new version available, update your EteSync server installation.
+To get the latest release with ``git``, you first have to commit the changes you made to ``~/etesync_server/etesync_server/settings.py``:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd ~/etesync_server
+ [isabell@stardust etesync_server]$ git add etesync_server/settings.py
```
[the remark this answers is not in the document]
Comment (Nepoh) [reply source/guide_etesync-server.rst]: Updated

File: source/guide_etesync-server.rst [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+  wsgi-file = %(base)/wsgi.py
+  touch-reload = %(wsgi-file)
+  static-map = /static=%(base)/static
+
+  app = wsgi
+
+  plugin = python
+
+  uid = <username>
+  gid = <username>
```
[the remark this answers is not in the document]
Comment (Nepoh) [reply source/guide_etesync-server.rst]: Updated

**Comment by Nepoh** [comment]
Glad you like it, this is my first guide and i really enjoyed writing it :)
Thanks for your additions, I applied them with the latest commit.

**Comment by luto** [comment]
nice! thank you :tada:

**Comment by SeanPedersen** [comment]
Hey just tried installing the latest version of etesync-server using this indeed nicely written guide but I failed to get the latest version of etesync-server running (I think etesync changed some configuration stuff and uwsgi interfaces, not sure though). If anyone updates it to the latest version of etesync and python and get's it running, I would be very interested in trying it out!

**Comment by Nepoh** [comment]
Happy to see the guide is in use! Etesync server from version 0.5.0 up is called "v2" and seems to introduce some major changes. They also changed the protocol used for communication with the clients, that's why I personally have not updated so far.
Unless someone wants to update the guide I created another pull request #864 to inform about the fact that it no longer works for the latest releases.

**Pull request merged**